src/plugins/discover/server/ui_settings.ts

```diff
@@ -303,4 +308,26 @@ export const getUiSettings: (docLinks: DocLinksServiceSetup) => Record<string, U
     schema: schema.number({ min: 0 }),
     requiresPageReload: true,
   },
+  [ENABLE_SQL]: {
+    name: i18n.translate('discover.advancedSettings.enableSQLTitle', {
+      defaultMessage: 'Enable SQL',
+    }),
+    value: false,
+    description: i18n.translate('discover.advancedSettings.enableSQLDescription', {
+      defaultMessage:
+        '{technicalPreviewLabel} This tech preview feature is highly experimental--do not rely on this for production saved searches or dashboards. This setting enables SQL as a text-based query language in Discover. If you have feedback on this experience please reach out to us on {link}',
```

Comment:
In other places in the UI, we use a "Technical preview" label with a tooltip.  Here is a screenshot with our standard wording.  Can we do something like that here?

![image (15)](https://user-images.githubusercontent.com/33642766/180327492-786a775b-9e89-43e5-aede-3cfc0ef9f1e7.png)

The setting label could then be :

**SQL queries in Discover**

Enable SQL as a text-based query language in Discover.  Do not use for saved searches and dashboards in a production environment. Provide feedback on our [community page](https://discuss.elastic.co/c/elastic-stack/kibana/7).

Reply:
I am afraid I can't add a tooltip there Gail. It is an implementation limitation.